core/driver/sources/src/main/resources/reference.conf

```diff
@@ -5,23 +5,35 @@ probe {
 
   // `intellij` is for org.virtuslab.ideprobe.config.IntellijConfig - default values below:
   intellij {
+
     plugins = []
+
     version {
+
       // `release` is the "Version" and connected `build` is the "Build Number" of IntelliJ IDEA version. Both should
       // be provided and they should match. You can search for proper string pairs here:
       // https://www.jetbrains.com/intellij-repository/releases/ under "com.jetbrains.intellij.idea" heading.
       // If you want to change the default intellij version, provide both parameters in your .conf file.
       build = "212.5080.55"
       release = "2021.2.1"
+
+      // `format` stands for the file format (file extension). You can choose, which type should be downloaded.
+      // By default, ".zip" distributions of IntelliJ IDEA are downloaded. Non-default supported format is ".dmg".
+      // Other formats are not supported. You can choose only one format.
+      format = ".zip"
+      # format = ".dmg"
     }
-    // Use below config with proper path if you want to use an existing Intellij instead of downloading a version.
+
+    // Use `path` config with proper path if you want to use an existing Intellij instead of downloading a version.
```

Comment:
Pls mention what's the relation b/w this config key, and `file:///...` entries in `resolvers.intellij.repositories`... which one takes precedence? Actually, `intellij.path` is somewhat redundant, given that you can specify a `file:///` path in `resolvers.intellij.repositories`, amirite?

Reply:
`probe.intellij.path` takes precedence. 

For example, if you would specify two different paths under `probe.intellij.path` and `probe.resolvers.intellij.repositories` - then the first one would be used to install IntelliJ. Using `probe.intellij.path` tells `ide-probe` to use an `org.virtuslab.ideprobe.ide.intellij.ExistingIntelliJ` instance instead of the `org.virtuslab.ideprobe.ide.intellij.IntelliJFactory`. This means that there will be no real usage of `probe.resolvers.intellij.repositories` value in such case. I find it rather unlikely that somebody would do such a thing in their `.conf` file, but I will add a comment about such case.

In my opinion - we can leave both configs for now and possibly clean this mess in some future issue. It would be nice to have only one config where a path to existing IntelliJ instance can be passed. Currently it is somehow confusing. 

The `probe.intellij.path` way is more obvious and could be used for simple cases, where someone would like to use **only** an existing IntelliJ instance for tests.

The `probe.resolvers.intellij.repositories` way is more complicated, but gives more possibilities - like in current git-machete-intellij-plugin solution (first - try to use an existing intelliJ and if not possible - download from the internet).